e2e/cypress/tests/integration/channel/leave_and_archive_channel_destructive_spec.ts

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {UserProfile} from 'mattermost-redux/types/users';
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @channel
+
+describe('Leave and Archive channel actions display as destructive', () => {
+    let testUser: UserProfile;
+    let offTopicUrl: string;
+
+    before(() => {
+        // # Login as test user and visit off-topic channel
+        cy.apiInitSetup().then(({user, offTopicUrl: url}) => {
+            testUser = user;
+            offTopicUrl = url;
+
+            cy.apiLogin(testUser);
+            cy.visit(offTopicUrl);
+        });
+    });
+
+    it('MM-T4943_1 Leave and Archive channel actions display as destructive in the channel dropdown menu', () => {
+        // # click on channel drop-down menu
+        cy.get('#channelHeaderTitle').should('be.visible').click();
+
+        // // * View Info menu option should be visible;
```

Comment:
```suggestion
        // * View Info menu option should be visible;
```

Can you remove double // , from rest of the comments